### PR TITLE
Engy branch: add tool links to ref-based transcriptomics tutorial

### PR DESCRIPTION
**Pull request by EngyNasr**
I have updated the ref-based tutorial (tools links) as per the comments on the last pull request on my_new_branch

I can created this branch instead, since I have got multiple errors on the last one that I couldnot resolve after many trials

Last comment: some tools e.g. filter, their usegalaxy tool links are different (without the toolshred)

### CODE REVIEW

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -142,12 +142,12 @@ Sequence quality control is therefore an essential first step in your analysis.
...
 > ### {% icon hands_on %} Hands-on: Quality control
 >
-> 1. **FastQC** {% icon tool %} with the following parameters:
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} with the following parameters:
 >    - {% icon param-files %} *"Short read data from your current history"*: input datasets selected with **Multiple datasets**
 >
 >    {% snippet faqs/galaxy/tools_select_multiple_datasets.md %}
 >
-> 2. Inspect the webpage output of **FastQC** {% icon tool %} for the `GSM461177` sample
+> 2. Inspect the webpage output of {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} for the `GSM461177` sample
```
[the remark this answers is not in the document]
Comment (bebatut) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md:150]: When the tool is just referenced and not used directly, the link is not needed. Could you change that everywhere? Thanks a lot

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 2]
```diff
@@ -322,10 +322,10 @@ We will map our reads to the *Drosophila melanogaster* genome using **STAR** ({%
 >    >
 >    {: .comment}
 >
-> 2. **RNA STAR** {% icon tool %} with the following parameters to map your reads on the reference genome:
+> 2. {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %} with the following parameters to map your reads on the reference genome:
 >    - *"Single-end or paired-end reads"*: `Paired-end (as individual datasets)`
->       - {% icon param-files %} *"RNA-Seq FASTQ/FASTA file, forward reads"*: the `Read 1 Output` (outputs of **Cutadapt** {% icon tool %})
->       - {% icon param-files %} *"RNA-Seq FASTQ/FASTA file, reverse reads"*: the `Read 2 Output` (outputs of **Cutadapt** {% icon tool %})
+>       - {% icon param-files %} *"RNA-Seq FASTQ/FASTA file, forward reads"*: the `Read 1 Output` (outputs of {% tool [Cutadapt](toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/1.16.5) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:327]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 3]
```diff
@@ -334,12 +334,12 @@ We will map our reads to the *Drosophila melanogaster* genome using **STAR** ({%
...
 >               This parameter should be length of reads - 1
 >
-> 3. **MultiQC** {% icon tool %} to aggregate the STAR logs:
+> 3. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.7) %} to aggregate the STAR logs:
 >      - In *"Results"*
 >        - *"Which tool was used generate logs?"*: `STAR`
 >        - In *"STAR output"*
 >           - *"Type of STAR output?"*: `Log`
->           - {% icon param-files %} *"STAR output"*: `log` files (output of **RNA STAR** {% icon tool %})
+>           - {% icon param-files %} *"STAR output"*: `log` files (output of {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:342]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 4]
```diff
@@ -452,15 +452,15 @@ The BAM file contains information for all our reads, making it difficult to insp
...
+> >    - {% icon param-files %} *"Select SAM/BAM dataset or dataset collection"*: `mapped.bam` files (outputs of {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %})
 > >
-> > 2. **MultiQC** {% icon tool %} to aggregate the MarkDuplicates logs:
+> > 2. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.7) %} to aggregate the MarkDuplicates logs:
 > >    - In *"Results"*
 > >      - *"Which tool was used generate logs?"*: `Picard`
 > >      - In *"Picard output"*
 > >         - *"Type of Picard output?"*: `Markdups`
-> >         - {% icon param-files %} *"Picard output"*: `MarkDuplicate metrics` files (output of **MarkDuplicates** {% icon tool %})
+> >         - {% icon param-files %} *"Picard output"*: `MarkDuplicate metrics` files (output of {% tool [MarkDuplicates](toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/2.18.2.2) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:463]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 5]
```diff
@@ -452,15 +452,15 @@ The BAM file contains information for all our reads, making it difficult to insp
 >
 > > ### {% icon hands_on %} Hands-on: Check duplicate reads
 > >
-> > 1. **MarkDuplicates** {% icon tool %}:
-> >    - {% icon param-files %} *"Select SAM/BAM dataset or dataset collection"*: `mapped.bam` files (outputs of **RNA STAR** {% icon tool %})
+> > 1. {% tool [MarkDuplicates](toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/2.18.2.2) %}:
+> >    - {% icon param-files %} *"Select SAM/BAM dataset or dataset collection"*: `mapped.bam` files (outputs of {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:456]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 6]
```diff
@@ -550,16 +550,16 @@ The BAM file contains information for all our reads, making it difficult to insp
 >
 > > ### {% icon hands_on %} Hands-on: Check the number of reads mapped to each chromosome
 > >
-> > 1. **Read Distribution** {% icon tool %}:
-> >    - {% icon param-files %} *"Input .bam/.sam file"*: `mapped.bam` files (outputs of **RNA STAR** {% icon tool %})
-> >    - *"Reference gene model"*: BED12 file (output **Convert GTF to BED12** {% icon tool %})
+> > 1. {% tool [Read Distribution](toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_read_distribution/2.6.4.1) %}:
+> >    - {% icon param-files %} *"Input .bam/.sam file"*: `mapped.bam` files (outputs of {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:554]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 7]
```diff
@@ -550,16 +550,16 @@ The BAM file contains information for all our reads, making it difficult to insp
...
+> >    - *"Reference gene model"*: BED12 file (output {% tool [Convert GTF to BED12](toolshed.g2.bx.psu.edu/repos/iuc/gtftobed12/gtftobed12/357) %})
 > >
-> > 2. **MultiQC** {% icon tool %} to aggregate the idxstats logs:
+> > 2. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.7) %} to aggregate the idxstats logs:
 > >    - In *"Results"*
 > >      - *"Which tool was used generate logs?"*: `RSeQC`
 > >      - In *"RSeQC output"*
 > >         - *"Type of RSeQC output?"*: `read_distribution`
-> >         - {% icon param-files %} *"RSeQC read_distribution output"*: outputs of **Read Distribution** {% icon tool %})
+> >         - {% icon param-files %} *"RSeQC read_distribution output"*: outputs of {% tool [Read Distribution](toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_read_distribution/2.6.4.1) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:562]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 8]
```diff
@@ -651,17 +651,17 @@ Another option is to estimate these parameters with a tool called **Infer Experi
...
 >
-> 1. **Convert GTF to BED12** {% icon tool %} to convert the GTF file to BED:
+> 1. {% tool [Convert GTF to BED12](toolshed.g2.bx.psu.edu/repos/iuc/gtftobed12/gtftobed12/357) %} to convert the GTF file to BED:
 >    - {% icon param-file %} *"GTF File to convert"*: `Drosophila_melanogaster.BDGP6.87.gtf`
 >
-> 2. **Infer Experiment** {% icon tool %} to determine the library strandness with:
->    - {% icon param-files %} *"Input .bam file"*: `mapped.bam` files (outputs of **RNA STAR** {% icon tool %})
->    - {% icon param-file %} *"Reference gene model"*: BED12 file (output **Convert GTF to BED12** {% icon tool %})
+> 2. {% tool [Infer Experiment](toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_infer_experiment/2.6.4.1) %} to determine the library strandness with:
+>    - {% icon param-files %} *"Input .bam file"*: `mapped.bam` files (outputs of {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:658]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 9]
```diff
@@ -716,8 +716,8 @@ We now run **featureCounts** to count the number of reads per annotated gene.
 
 > ### {% icon hands_on %} Hands-on: Counting the number of reads per annotated gene
 >
-> 1. **featureCounts** {% icon tool %} to count the number of reads per gene:
->    - {% icon param-files %} *"Alignment file"*: `mapped.bam` files (outputs of **RNA STAR** {% icon tool %})
+> 1. {% tool [featureCounts](toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts/2.0.1) %} to count the number of reads per gene:
+>    - {% icon param-files %} *"Alignment file"*: `mapped.bam` files (outputs of {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:720]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 10]
```diff
@@ -731,10 +731,10 @@ We now run **featureCounts** to count the number of reads per annotated gene.
 >       - *"Allow reads to map to multiple features"*: `Disabled; reads that align to multiple features or overlapping features are excluded`
 >       - *"Minimum mapping quality per read"*: `10`
 >
-> 2. **MultiQC** {% icon tool %} to aggregate the report:
+> 2. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.7) %} to aggregate the report:
 >     - In *"Results"*:
 >       - *"Which tool was used generate logs?"*: `featureCounts`
->           - {% icon param-files %} *"Output of FeatureCounts"*: `summary` files (outputs of **featureCounts** {% icon tool %})
+>           - {% icon param-files %} *"Output of FeatureCounts"*: `summary` files (outputs of {% tool [featureCounts](toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts/2.0.1) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:737]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 11]
```diff
@@ -1275,7 +1275,7 @@ Now we would like to extract the most differentially expressed genes due to the
 
 > ### {% icon hands_on %} Hands-on: Extract the most differentially expressed genes
 >
-> 1. **Filter data on any column using simple expressions** {% icon tool %} to extract genes with a significant change in gene expression (adjusted *p*-value below 0.05) between treated and untreated samples:
+> 1. {% tool [Filter data on any column using simple expressions](https://usegalaxy.eu/root?tool_id=Filter1) %} to extract genes with a significant change in gene expression (adjusted *p*-value below 0.05) between treated and untreated samples:
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1278]:
```suggestion
> 1. {% tool [Filter data on any column using simple expressions](Filter1) %} to extract genes with a significant change in gene expression (adjusted *p*-value below 0.05) between treated and untreated samples:
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 12]
```diff
@@ -1299,7 +1299,7 @@ Now we would like to extract the most differentially expressed genes due to the
 >
 >    We will now select only the genes with an absolute fold change (FC) > 2
 >
-> 3. **Filter** {% icon tool %} to extract genes with an $$abs(log_{2} FC) > 1$$:
+> 3. {% tool [Filter](https://usegalaxy.eu/root?tool_id=Filter1) %} to extract genes with an $$abs(log_{2} FC) > 1$$:
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1302]:
```suggestion
> 3. {% tool [Filter](Filter1) %} to extract genes with an $$abs(log_{2} FC) > 1$$:
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 13]
```diff
@@ -1406,8 +1406,8 @@ We proceed in several steps:
 To extract the normalized counts for the interesting genes, we join the normalized count table generated by DESeq2 with the table we just generated. We will then keep only the lines corresponding to the most differentially expressed genes.
 
 > ### {% icon hands_on %} Hands-on: Extract the normalized counts of the most differentially expressed genes
-> 1. **Join two Datasets** {% icon tool %} with the following parameters:
->    - {% icon param-file %} *"Join"*: the `Normalized counts` file (output of **DESeq2** {% icon tool %})
+> 1. {% tool [Join two Datasets](toolshed.g2.bx.psu.edu/repos/devteam/join/gops_join_1/1.0.0) %} with the following parameters:
+>    - {% icon param-file %} *"Join"*: the `Normalized counts` file (output of {% tool [DESeq2](toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2/2.11.40.6+galaxy1) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1410]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 14]
```diff
@@ -1416,10 +1416,10 @@ To extract the normalized counts for the interesting genes, we join the normaliz
 >
 >    The generated file has more columns than we need for the heatmap: mean normalized counts, $$log_{2} FC$$ and other annotation information. We need to remove the extra columns.
 >
-> 2. **Cut** {% icon tool %} to extract the columns with the gene IDs and normalized counts:
+> 2. {% tool [Cut](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cut_tool/1.1.0) %} to extract the columns with the gene IDs and normalized counts:
 >    - *"Cut columns"*: `c1-c8`
 >    - *"Delimited by"*: `Tab`
->    - {% icon param-file %} *"From"*: the joined dataset (output of **Join two Datasets** {% icon tool %})
+>    - {% icon param-file %} *"From"*: the joined dataset (output of {% tool [Join two Datasets](toolshed.g2.bx.psu.edu/repos/devteam/join/gops_join_1/1.0.0) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1422]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 15]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
+> 1. {% tool [Compute](toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.6) %} with
 >    - *"Add expression"*: `bool(c7<0.05)`
 >    - {% icon param-file %} *"as a new column to"*: the `DESeq2 result file`
 >
-> 2. **Cut** {% icon tool %} with
+> 2. {% tool [Cut](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cut_tool/1.1.0) %} with
 >    - *"Cut columns"*: `c1,c8`
 >    - *"Delimited by"*: `Tab`
->    - {% icon param-file %} *"From"*: the output of the **Compute** {% icon tool %}
+>    - {% icon param-file %} *"From"*: the output of the {% tool [Compute](toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.6) %}
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1563]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 16]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
 >
-> 3. **Change Case** {% icon tool %} with
->    - {% icon param-file %} *"From"*: the output of the previous **Cut** {% icon tool %}
+> 3. {% tool [Change Case](https://usegalaxy.eu/root?tool_id=ChangeCase) %} with
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1565]:
```suggestion
> 3. {% tool [Change Case](ChangeCase) %} with
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 17]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
+>    - {% icon param-file %} *"From"*: the output of the previous {% tool [Cut](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cut_tool/1.1.0) %}
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1566]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 18]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
 >    - *"Change case of columns"*: `c1`
 >    - *"Delimited by"*: `Tab`
 >    - *"To"*: `Upper case`
 >
 > 4. Rename the output to `Gene IDs and differentially expression`
 >
->    We just generated the first input for **goseq**. As second input for **goseq** we need the gene lengths. We can use here the gene lengths generated by **featureCounts** {% icon tool %} and format the gene IDs.
+>    We just generated the first input for **goseq**. As second input for **goseq** we need the gene lengths. We can use here the gene lengths generated by {% tool [featureCounts](toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts/2.0.1) %} and format the gene IDs.
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1573]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 19]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
 >
-> 5. Drag and drop one of the feature length datasets generated by **featureCounts** {% icon tool %} into this history using the {% icon galaxy-columns %} **View all histories**
+> 5. Drag and drop one of the feature length datasets generated by {% tool [featureCounts](toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts/2.0.1) %} into this history using the {% icon galaxy-columns %} **View all histories**
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1575]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 20]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
 >
-> 6. **Change Case** {% icon tool %} with
->    - {% icon param-file %} *"From"*: the `feature lengths` (output of **featureCounts** {% icon tool %})
+> 6. {% tool [Change Case](https://usegalaxy.eu/root?tool_id=ChangeCase) %} with
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1577]:
```suggestion
> 6. {% tool [Change Case](ChangeCase) %} with
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 21]
```diff
@@ -1553,29 +1553,29 @@ We have extracted genes that are differentially expressed in treated (PS gene-de
...
+>    - {% icon param-file %} *"From"*: the `feature lengths` (output of {% tool [featureCounts](toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts/2.0.1) %})
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1578]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 22]
```diff
@@ -1625,11 +1625,11 @@ We have now the two required input files for goseq.
     > >
     > > 1. 31 GO terms (0.27%) are over-represented and 83 (0.72%) under-represented
     > >
-    > >    **Filter** {% icon tool %} on c8 (adjusted p-value for over-represented GO terms) and c9 (adjusted p-value for under-represented GO terms)
+    > >    {% tool [Filter](https://usegalaxy.eu/root?tool_id=Filter1) %} on c8 (adjusted p-value for over-represented GO terms) and c9 (adjusted p-value for under-represented GO terms)
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1628]:
```suggestion
    > >    {% tool [Filter](Filter1) %} on c8 (adjusted p-value for over-represented GO terms) and c9 (adjusted p-value for under-represented GO terms)
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 23]
```diff
@@ -1625,11 +1625,11 @@ We have now the two required input files for goseq.
...
     > >
     > > 2. For over-represented, 20 BP, 3 CC and 8 MF and for under-represented, 49 BP, 29 CC and 5 MF
     > >
-    > >    **Group data** {% icon tool %} on column 7 (category) and count on column 1 (IDs)
+    > >    {% tool [Group data](https://usegalaxy.eu/root?tool_id=Grouping1) %} on column 7 (category) and count on column 1 (IDs)
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1632]:
```suggestion
    > >    {% tool [Group data](Grouping1) %} on column 7 (category) and count on column 1 (IDs)
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 24]
```diff
@@ -1693,9 +1693,9 @@ As for the GO terms, 2 files are generated:
     > > ### {% icon solution %} Solution
     > >
     > > 1. The file has 127 lines including an header, so 126 KEGG pathways have been identified.
-    > > 2. 2 KEGG pathways (1.57%) are over-represented, using **Filter** {% icon tool %} on c6 (adjusted p-value for over-represented KEGG pathways)
+    > > 2. 2 KEGG pathways (1.57%) are over-represented, using {% tool [Filter](https://usegalaxy.eu/root?tool_id=Filter1) %} on c6 (adjusted p-value for over-represented KEGG pathways)
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1696]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 25]
```diff
@@ -1693,9 +1693,9 @@ As for the GO terms, 2 files are generated:
...
     > > 3. The first 2 KEGG pathways are `01100` and `00010`. By searching on the [KEGG database](https://www.genome.jp/kegg/kegg2.html) for them, we can find more information about these pathways: `01100` corresponds to all metabolic pathways and `00010` to pathway for Glycolysis / Gluconeogenesis.
-    > > 4. 1 KEGG pathways (0.79%) is under-represented, using **Filter** {% icon tool %} on c7 (adjusted p-value for under-represented KEGG pathways): `03040`, Spliceosome
+    > > 4. 1 KEGG pathways (0.79%) is under-represented, using {% tool [Filter](https://usegalaxy.eu/root?tool_id=Filter1) %} on c7 (adjusted p-value for under-represented KEGG pathways): `03040`, Spliceosome
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1698]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 26]
```diff
@@ -1782,16 +1782,16 @@ This step is similar to the step of [counting the number of reads per annotated
...
 >       - {% icon param-file %} *"GTF file"*: `Drosophila_melanogaster.BDGP6.87.gtf`
 >
 >    The output is again a GTF file that is ready to be used for counting
 >
-> 2. **DEXSeq-Count** {% icon tool %}: Count reads using **DEXSeq-Count** with
+> 2. {% tool [DEXSeq-Count](toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq_count/1.28.1.0) %}: Count reads using **DEXSeq-Count** with
 >     - *"Mode of operation"*: `Count reads`
->       - {% icon param-files %} *"Input bam file"*: the `BAM` files generated by **RNA STAR** {% icon tool %}
->       - {% icon param-file %} *"DEXSeq compatible GTF file"*: the GTF file generated by **DEXSeq-Count** {% icon tool %}
+>       - {% icon param-files %} *"Input bam file"*: the `BAM` files generated by {% tool [RNA STAR](toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) %}
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1793]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 27]
```diff
@@ -1782,16 +1782,16 @@ This step is similar to the step of [counting the number of reads per annotated
...
+>       - {% icon param-file %} *"DEXSeq compatible GTF file"*: the GTF file generated by {% tool [DEXSeq-Count](toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq_count/1.28.1.0) %}
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1794]: Not need for link here

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 28]
```diff
@@ -1893,7 +1893,7 @@ Similarly to DESeq2, DEXSeq generates a table with:
 
 > ### {% icon hands_on %} Hands-on:
 >
-> 1. **Filter** {% icon tool %} to extract exons with a significant differential usage (adjusted *p*-value equal or below 0.05) between treated and untreated samples
+> 1. {% tool [Filter](https://usegalaxy.eu/root?tool_id=Filter1) %} to extract exons with a significant differential usage (adjusted *p*-value equal or below 0.05) between treated and untreated samples
```
Comment (bebatut) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1896]:
```suggestion
> 1. {% tool [Filter](Filter1) %} to extract exons with a significant differential usage (adjusted *p*-value equal or below 0.05) between treated and untreated samples
```

**Review by bebatut — APPROVED** [review]
Thanks @EngyNasr for your first contribution there!

**Comment by shiltemann** [comment]
Whoo! Thanks a lot @EngyNasr!

**Pull request merged**